client/src/Video.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from "react";
+import Button from "react-bootstrap/Button";
+import axios from "axios";
+
+function Video({ info }) {
+  const [likeCount, setLikeCount] = useState(info.rating);
+
+  const handleLike = (status, id) => {
+    let newRate = likeCount;
+    if (status === "like") {
+      newRate = newRate + 1;
+      setLikeCount(newRate);
+    } else {
+      newRate = newRate - 1;
+      setLikeCount(newRate);
+    }
+    try {
+      axios.put(`https://full-stack-assessment.onrender.com/video/${id}`, {
+        newRate,
+      });
+    } catch (err) {
+      console.log(err);
+    }
+  };
+
+  const handleDelete = (id) => {
+    axios.delete(`https://full-stack-assessment.onrender.com/video/${id}`);
+    alert("video has been deleted");
+    window.location.reload();
```

Comment:
Try updating your videos list in React state, to avoid having to do a full page reload.